common/static/common/js/spec_helpers/discussion_spec_helper.js

```diff
@@ -53,7 +53,7 @@
         DiscussionSpecHelper.makeCourseSettings = function() {
             return new DiscussionCourseSettings({
                 category_map: {
-                    children: ['Test Topic', 'Other Topic'],
+                    children: [['Test Topic', 'entry'], ['Other Topic', 'entry']],
```

Comment:
@robrap Thank you for following up on the suggestion, I agree that would require a lot of code changes which will go beyond the scope of this change. As it is not the requirement for this change I agree we should move forward as-is.